### ISSUE TITLE
Filter-column strange behaviour on chrome 

### ISSUE THREAD

**Issue by rmarzocchi84** [wenzhixin/bootstrap-table#4816]
I noted a strange behaviour of filter-column option on google chrome, not in firefox. 

I have a table which use filter column. In chrome the input filter do not works correctly, while in firefox it works!

Here a video which explain the behaviour:

![chrome_vs_firefox](https://user-images.githubusercontent.com/4061154/73351465-53171680-428f-11ea-8368-01b66edd3e26.gif)

My code is:

```
<div style="overflow-x:auto;">  
  <table  id="segnalazioni" class="table-hover" data-toggle="table" data-filter-control="true"
  data-show-search-clear-button="true"
  data-url="./tables/griglia_segnalazioni.php?r=&f=&from=&to=&m="
  data-show-export="true" data-search="true" data-click-to-select="true" data-pagination="true"
  data-sidePagination="true" data-show-refresh="true" data-show-toggle="true" data-show-columns="true" data-toolbar="#toolbar">
  <thead>
  <tr>
  <th data-field="state" data-checkbox="true"></th>
  <th data-field="id" data-sortable="true" data-formatter="nameFormatterEdit" data-visible="true" data-filter-control="input">Dettagli</th>
  <th data-field="in_lavorazione" data-sortable="true" data-formatter="nameFormatter" data-visible="true" data-filter-control="select">Stato</th>
  <th data-field="rischio" data-sortable="true" data-formatter="nameFormatterRischio" data-visible="true">Persone<br>a rischio</th>
  <th data-field="criticita" data-sortable="true"   data-visible="true" data-filter-control="select">Tipo<br>criticità</th>
  <th data-field="data_ora" data-sortable="true"  data-visible="true">Data e ora</th>
  <th data-field="descrizione" data-sortable="true"  data-visible="true">Descrizione</th>
  <th data-field="nome_munic" data-sortable="true"  data-visible="true" data-filter-control="select">Municipio</th>
  <th data-field="localizzazione" data-sortable="true"  data-visible="true" data-filter-control="input" >Civico</th>
   <th data-field="id2" data-sortable="false" data-formatter="nameFormatterMappa1" data-visible="true" Anteprima<br>mappa</th>
  <th data-field="note" data-sortable="false" data-visible="true" >Note</th>
   <th data-field="id_evento" data-sortable="true"  data-visible="true" data-filter-control="select">Id<br>evento</th>
   <th data-field="tipo_evento" data-sortable="true"  data-visible="true" data-filter-cntrol="select">Tipo<br>evento</th>
   
  </tr>
  </thead>
  
  </table>
 </div>
```

**Comment by djhvscf** [comment]
Hi, thanks for your report. Please provide an [Online Example](https://live.bootstrap-table.com/) to show your problem, thanks!

**Comment by rmarzocchi84** [comment]
Something similar to this example https://live.bootstrap-table.com/code/rmarzocchi84/1617 but it works correctly

**Comment by djhvscf** [comment]
Which version of Bootstrap-table are you using? Which extensions are you using besides Filter-Control?

**Comment by rmarzocchi84** [comment]
I have some news: 
- the problem happens only on Chrome 79.0.3945.130 (OS Windows 10) while on Chrome 70.0.3538.102 it works correctly (like on firefox)
- the problem happens using a master version updated two weeks ago. I have done now a 
`git checkout tags/1.15.5`
and it works also on Chrome 79.0.3945.130 (OS Windows 10)

Finally I use also the table-export extension

**Comment by djhvscf** [comment]
@wenzhixin

**Comment by wenzhixin** [comment]
Validation is required after a new version is released.

**Comment by UtechtDustin** [comment]
I close this issue because i cant reproduce the issue in the newest version of the bootstrap-table. If this issue still exists please provide us an example (using our editor).

**Comment by rmarzocchi84** [comment]
Sorry I encountered the same error on chrome using the json I put here https://gishosting.gter.it/test.json

I try to put data in this example https://live.bootstrap-table.com/code/rmarzocchi84/2567 but there is a problem in access to json data

**Comment by rmarzocchi84** [comment]
I have tested both v.1.15.5 and 1.16.0

with different data (different json it works fine also on chrome browser)

**Comment by UtechtDustin** [comment]
I will test it as soon i have time :-)

**Comment by rmarzocchi84** [comment]
Thanks @UtechtDustin .Unfortunately the bahviour is really strange.
I try to extract the table code here [1] https://gishosting.gter.it/test_table/pages/elenco_segnalazioni.php 
and it works correctly

Summarizing (in order to try to understand the reason of the problem):
- page complete it works fine on firefox, it works fine on chrome with some data (_json1_), it give problem with other data (_json2_)
- page extracted [1] it works fine also using _json2_ 

_json2_=https://gishosting.gter.it/test.json
_json1_=https://gishosting.gter.it/test1.json

**Comment by rmarzocchi84** [comment]
Sorry @UtechtDustin, I have finally solved using version 1.16.0 and **clearing the browser cache** and now it works also on the complete page. 

There is an other problem if I use font awasome symbol in the column  and `data-filter-control="select"`
You can see the problem here: https://gishosting.gter.it/test_table/pages/elenco_segnalazioni.php

I do not know if it is better to close this issue and open a different issue

![image](https://user-images.githubusercontent.com/4061154/79959896-ff2e5f00-8484-11ea-94fb-d6e27f4862ac.png)

**Comment by UtechtDustin** [comment]
@rmarzocchi84 
Thank you for the feedback.
Your current Issue has been fixed in #4955 ([Example](https://live.bootstrap-table.com/code/UtechtDustin/2426)).
It will be fixed in the next version (1.16.1).
